File: lonelyinteger/lonelyInteger.js
const a = [1, 2, 3, 4, 3, 2, 1]


function lonelyinteger(a) {
    let answer = 0

    for (i=0; i<a.length; i++){
        let counter = 0
        for (j=0; j<a.length; j++) {
            if (a[i]==a[j]) {
                counter +=1
            }
        }
        if (counter == 1) {
            answer = a[i]
        }

    }
    console.log(answer);
    return(answer)
}

lonelyinteger(a)

File: lonelyinteger/lonely_integer.py
import math
import os
import random
import re
import sys

a = [1, 2, 3, 4, 3, 2, 1]


def lonelyinteger(a):
    for i in a:
        counter = 0
        for j in a:
            if i == j:
                counter+=1
        if counter == 1:
            print(i)
            return(i)



lonelyinteger(a)

